src/components/FilterMenu/FilterMenuItemRange.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="filter-menu-item-range">
+    <SvgIcon
+      v-if="showIcon"
+      :name="iconName"
+      class="filter-menu-item-range__icon"
+    />
+
+    <input
+      :id="inputId"
+      v-model.number="spacesStore.filters[name]"
+      type="range"
+      min="0"
+      :max="maxValue"
+      class="filter-menu-item-range__input"
+    >
+
+    <label :for="inputId">
+      {{ labelValue ?? $t('noSeatsFilterLabel') }}
+      <span class="a11y-sr-only">
+        {{ label ?? $t(i18nKey) }}
+      </span>
+    </label>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useSpacesStore } from "~/stores/spaces";
+import type { Filters } from "~/types/Filters";
+
+const props = withDefaults(
+  defineProps<{
+    /** The name of the property to bind to in the global filter object */
+    name: keyof Filters;
+    /** Used to determine the i18n key for the labe and the icon name. If not specified, name is used */
+    displayKey?: string;
+    /** Allows to override the label text */
+    label?: string;
+    /** False if you do not want to display an icon */
+    showIcon?: boolean;
+    /** The maximum input value of the range slider */
+    maxValue?: number;
+  }>(),
+  {
+    displayKey: undefined,
+    label: undefined,
+    showIcon: true,
+    maxValue: 10,
+  }
+);
+
+const inputId = computed(() =>
+  `filter-item-${props.name}`
+);
+
+const i18nKey = computed(() =>
+  [props.displayKey ?? props.name].filter(Boolean).join(".")
+);
+
+const iconName = computed(() => `facility-${i18nKey.value}-icon`);
+
+const labelValue = computed(() => {
+  if (spacesStore.filters[props.name] == 0) {
+    return null
```

Comment:
Looks like the filter value can also be 0 as a valid number. But will be represented as null when its 0.